articles/cosmos-db/monitor-resource-logs.md

```diff
@@ -42,7 +42,7 @@ Here, we walk through the process of creating diagnostic settings for your accou
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
 
-1. Navigate to your Azure Cosmos DB account. Open the **Diagnostic settings** pane under the **Monitoring section**, and then select **Add diagnostic setting** option.
+1. Navigate to your Azure Cosmos DB account. Open the **Diagnostic settings** pane under the **Monitoring section**, click on the **Not now** button, and then select **Add diagnostic setting** option.
```

Comment:
Unfortunately, there's two main issues with this change:

1. This instruction tells you to "click a button" which is not inclusive terminology.
2. This instruction encourages the reader to skip an important warning message. This instruction assumes the reader will always see this warning, which is not always the case. This would be better served as a note block encouraging the reader to read the warning and giving them an option of clicking "not now."